src/ZoneServer/Buffs/Handlers/EnchantEarth_Buff.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Melia.Shared.Tos.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Scripting;
+using Melia.Zone.World.Actors.Characters;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Enchant Earth Buff, which increases the target's block penetration
+	/// </summary>
+	[BuffHandler(BuffId.EnchantEarth_Buff)]
+	public class EnchantEarth_Buff : BuffHandler
+	{
+		private float blockPenetrationBonus = 0;
+
+		public override void OnStart(Buff buff)
+		{
+			var target = buff.Target as Character;
+			var SCR_Get_Character_MAXPATK = ScriptableFunctions.Character.Get("SCR_Get_Character_MAXPATK");
```

Comment:
Unless you have a reason to directly access the scriptable function, you should use the property instead, as it's simpler and reduces the risk for errors, because the property getter will never change.